addons/resource.language.en_gb/resources/strings.po

```diff
@@ -2192,7 +2192,11 @@ msgctxt "#470"
 msgid "Credits"
 msgstr ""
 
-#empty strings from id 471 to 473
+msgctxt "#471"
+msgid "Disc Title"
```

Comment:
While it seems obvious to put this here near "disc" the 3-digit message text ids are old and impossible to search for, hence no new messages are to be added in this range.  Move to a higher ID